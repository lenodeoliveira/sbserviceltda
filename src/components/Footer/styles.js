import styled, { css } from 'styled-components'
import { Container as SectionContainer } from '../SectionContainer'

export const Container = styled.footer`
  ${({ theme }) => css`
    text-align: center;
    border-top: 0.1rem solid ${theme.colors.mediumGray};
    p {
      font-size: 1.8rem;
    }
    & ${SectionContainer} {
      padding-top: 0;
      padding-bottom: 0;
    }
  `}
`
