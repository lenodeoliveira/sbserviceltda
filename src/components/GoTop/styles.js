import styled, { css } from 'styled-components'

export const Container = styled.a`
  ${({ theme }) => css`
    position: fixed;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
    color: ${theme.colors.white};
    background: ${theme.colors.primaryColor};
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    opacity: 0.8;
  `}
`
