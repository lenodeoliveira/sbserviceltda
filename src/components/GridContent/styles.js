import styled, { css } from 'styled-components'

export const Container = styled.div`
  ${() => css`
    text-align: center;
    max-width: 58rem;
    margin: 0 auto;
  `}
`
export const TextContainer = styled.div`
  ${({ theme }) => css`
    margin: ${theme.spacings.xhuge} 0;
  `}
`
