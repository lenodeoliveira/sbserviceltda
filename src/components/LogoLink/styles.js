import styled, { css } from 'styled-components'

export const Container = styled.a`
  ${() => css`
    text-decoration: none;
    display: flex;
    align-items: center;
    color: inherit;
    > img {
      max-height: 2.5rem;
    }
  `}
`
