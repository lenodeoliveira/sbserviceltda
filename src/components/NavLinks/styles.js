import styled, { css } from 'styled-components'

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-flow: row wrap;
    background: red;

    @media ${theme.media.lteMedium} {
      flex-flow: column wrap;
      align-content: center;
    }
  `}
`
