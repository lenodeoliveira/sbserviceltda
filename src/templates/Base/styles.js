import styled, { css } from 'styled-components'

export const Container = styled.div`
  ${({ theme }) => css`
    padding-top: 2.4rem;
    @media ${theme.media.lteMedium} {
      padding-top: 0;
    }
  `}
`
