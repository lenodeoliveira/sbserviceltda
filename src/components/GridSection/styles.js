import styled, { css } from 'styled-components'
import { Container as TextComponent } from '../TextComponent/styles'
import { Title as Heading } from '../Heading/styles'

export const Container = styled.div`
  ${({ theme }) => css`
    ${TextComponent} {
      margin-bottom: ${theme.spacings.xhuge};
      margin-left: 7.3rem;
    }
    & > ${Heading} {
      display: flex;
      justify-content: center;
      margin-top: -10%;
    }
    @media ${theme.media.lteMedium} {
      & > ${Heading} {
        font-size: ${theme.font.sizes.large};
        margin-top: 0;
      }
      ${TextComponent} {
        margin-bottom: ${theme.spacings.xhuge};
        margin-left: 0;
      }
      text-align: center;
    }
  `}
`
export const Grid = styled.div`
  ${({ theme }) => css`
    display: grid;
    /* grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
    gap: ${theme.spacings.medium}; */
    grid-template-columns: 1fr 1fr 1fr 2fr;
    margin-left: 1.3rem;
    text-align: center;
    overflow: hidden;
    width: 100%;
    @media ${theme.media.lteMedium} {
      grid-template-columns: 1fr;
      margin-left: 0;
    }
  `}
`
export const GridElement = styled.div`
  ${({ theme }) => css`
    @media ${theme.media.lteMedium} {
      margin-bottom: ${theme.spacings.medium};
    }
  `}
`
