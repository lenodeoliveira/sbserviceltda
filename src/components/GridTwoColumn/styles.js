import styled, { css } from 'styled-components'
import { Title } from '../Heading/styles'

export const Container = styled.div`
  ${({ theme }) => css`
    display: grid;
    grid-template-columns: 1.8fr 1fr;
    margin-left: 3rem;
    align-items: center;
    gap: ${theme.spacings.large};

    @media ${theme.media.lteMedium} {
      grid-template-columns: 1fr;
      text-align: center;
      margin-left: 0;
    }
    ${Title} {
      margin-bottom: ${theme.spacings.xlarge};
    }
  `}
`
export const TextContainer = styled.div`
  ${({ theme }) => css`
    @media ${theme.media.lteMedium} {
      margin-bottom: ${theme.spacings.large};
    }
  `}
`
export const ImageContainer = styled.div`
  ${() => css`
    width: 100%;
  `}
`
export const Image = styled.img`
  ${() => css`
    width: 100%;
    border-radius: 2.4%;
  `}
`
