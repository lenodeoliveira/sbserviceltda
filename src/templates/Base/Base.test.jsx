import { screen } from '@testing-library/react'
import { Base } from '.'
import { renderTheme } from '../../styles/render-theme'

describe('<Base />', () => {
  it('Should render', () => {
    renderTheme(<Base>Children</Base>)
    expect(screen.getByRole('heading')).toBeInTheDocument()
  })
})
