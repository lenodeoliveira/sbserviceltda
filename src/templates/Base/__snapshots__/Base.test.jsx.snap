// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base /> Should render a Base component 1`] = `
.c11 {
  padding-top: 2.4rem;
}

.c4 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

.c2 {
  position: fixed;
  z-index: 5;
  top: 0;
  left: 0;
  right: 0;
  width: 100%;
  border-bottom: #DDDDDD;
  background: #FFFFFF;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c2 > .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c2 {
  margin-top: 0;
  margin-bottom: 0;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 {
  z-index: 6;
  position: fixed;
  top: 2rem;
  right: 2rem;
  width: 4rem;
  height: 4rem;
  background: #2F5233;
  color: #FFFFFF;
  border: none;
  display: none;
  pointer-events: all;
}

.c0 > svg {
  width: 2.5rem;
  height: 2.5rem;
}

.c8 {
  -webkit-text-decoration: none;
  text-decoration: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: inherit;
}

.c8 > img {
  max-height: 2.5rem;
}

.c7 {
  color: #2F5233;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c14 {
  color: #FFFFFF;
  font-size: 6.4rem;
  text-transform: none;
}

.c19 {
  color: #FFFFFF;
  font-size: 3.2rem;
  text-transform: none;
}

.c21 {
  color: #2F5233;
  font-size: 6.4rem;
  text-transform: none;
}

.c22 {
  color: #2F5233;
  font-size: 3.2rem;
  text-transform: none;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c10 {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #2F5233;
  position: relative;
}

.c10::after {
  content: '';
  position: absolute;
  bottom: 0.77rem;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #2F5233;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c10:hover::after {
  left: 25%;
  width: 50%;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c23 {
  text-align: center;
  color: #2F5233;
  border-top: 0.1rem solid #DDDDDD;
}

.c23 p {
  font-size: 1.8rem;
}

.c23 {
  padding-top: 0;
  padding-bottom: 0;
}

.c16 {
  font-size: 2.4rem;
}

.c24 {
  position: fixed;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
  color: #FFFFFF;
  background: #2F5233;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  opacity: 0.8;
}

.c13 .c15 {
  margin-bottom: 6.4rem;
  margin-left: 7.3rem;
}

.c13 > .c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-top: -10%;
}

.c17 {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 2fr;
  margin-left: 1.3rem;
  text-align: center;
  overflow: hidden;
  width: 100%;
}

.c12 {
  background: #FFFFFF;
  color: #2F5233;
  background: #2F5233;
  color: #FFFFFF;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c20 {
  background: #FFFFFF;
  color: #2F5233;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (max-width:768px) {
  .c11 {
    padding-top: 0;
  }
}

@media (max-width:768px) {
  .c2 {
    height: 100vh;
    visibility: hidden;
    opacity: 0;
  }

  .c2 > .c3 {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 1fr;
    height: 100vh;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    overflow-y: auto;
  }

  .c2 {
    padding-bottom: 3.2rem;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c5 {
    display: block;
    text-align: center;
    padding: 4.8rem 0;
  }
}

@media (max-width:768px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c14 {
    font-size: (max-width:768px);
  }
}

@media (max-width:768px) {
  .c21 {
    font-size: (max-width:768px);
  }
}

@media (max-width:768px) {
  .c9 {
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
  }
}

@media (max-width:768px) {
  .c13 {
    text-align: center;
  }

  .c13 > .c6 {
    font-size: 3.2rem;
    margin-top: 0;
  }

  .c13 .c15 {
    margin-bottom: 6.4rem;
    margin-left: 0;
  }
}

@media (max-width:768px) {
  .c17 {
    grid-template-columns: 1fr;
    margin-left: 0;
  }
}

@media (max-width:768px) {
  .c18 {
    margin-bottom: 2.4rem;
  }
}

<div>
  <button
    aria-label="Open/Close menu"
    class="c0"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    class="c2"
  >
    <div
      class="c3 c4"
    >
      <div
        class="c5"
      >
        <h1
          class="c6 c7"
        >
          <a
            class="c8"
            href="#"
          >
            Logo
          </a>
        </h1>
        <nav
          aria-label="Main menu"
          class="c9"
        >
          <a
            class="c10"
            href="#target1"
            target="_self"
          >
            Link 1
          </a>
          <a
            class="c10"
            href="#target2"
            target="_self"
          >
            Link 2
          </a>
          <a
            class="c10"
            href="#target3"
            target="_self"
          >
            Link 3
          </a>
          <a
            class="c10"
            href="#target4"
            target="_self"
          >
            Link 4
          </a>
          <a
            class="c10"
            href="#target4"
            target="_self"
          >
            Link 4
          </a>
          <a
            class="c10"
            href="#target5"
            target="_self"
          >
            Link 5
          </a>
          <a
            class="c10"
            href="#target6"
            target="_self"
          >
            Link 6
          </a>
          <a
            class="c10"
            href="#target7"
            target="_self"
          >
            Link 7
          </a>
          <a
            class="c10"
            href="#target8"
            target="_self"
          >
            Link 8
          </a>
          <a
            class="c10"
            href="#target9"
            target="_self"
          >
            Link 9
          </a>
          <a
            class="c10"
            href="#target10"
            target="_self"
          >
            Link 10
          </a>
        </nav>
      </div>
    </div>
  </div>
  <div
    class="c11"
  >
    <div
      class="c12"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <p
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </p>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 1
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 2
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 3
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 3
              </h3>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c20"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c21"
          >
            My grid
          </h2>
          <p
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </p>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 1
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 2
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 3
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 3
              </h3>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <p
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </p>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 1
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 2
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 3
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 3
              </h3>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c20"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c21"
          >
            My grid
          </h2>
          <p
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </p>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 1
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 2
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 3
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 3
              </h3>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c12"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c14"
          >
            My grid
          </h2>
          <p
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </p>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 1
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 2
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 3
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c19"
              >
                Teste 3
              </h3>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c20"
      id=""
    >
      <div
        class="c3 c4"
      >
        <div
          class="c13"
        >
          <h2
            class="c6 c21"
          >
            My grid
          </h2>
          <p
            class="c15 c16"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </p>
          <div
            class="c17"
          >
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 1
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 2
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 3
              </h3>
            </div>
            <div
              class="c18"
            >
              <h3
                class="c6 c22"
              >
                Teste 3
              </h3>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="c23"
    >
      <div
        class="c3 c4"
      >
        <p
          class="c15 c16"
        >
          SB Serviços de Radiologia e Telerradiologia 2022
           
        </p>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="c24"
    href="#"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
